packages/framework/store/src/store/doc/block.ts

```diff
@@ -285,7 +293,13 @@ export class Block {
           typeof p === 'string' &&
           model.keys.includes(p)
         ) {
-          this.yBlock.delete(`prop:${p}`);
+          if (this.doc) {
```

Comment:
What if a transaction need to include multiple operations? This doesn't make sense.